bokehjs/test/unit/models/formatters/log_tick_formatter.ts

```diff
@@ -8,19 +8,19 @@ describe("FuncTickFormatter", () => {
     it("should format numerical ticks appropriately", () => {
       const formatter = new LogTickFormatter()
       const labels = formatter.doFormat([0.001, 0.1, 1, 10, 100], {loc: 0})
-      expect(labels).to.be.equal(["10^-3", "10^-1", "10^0", "10^1", "10^2"])
+      expect(labels).to.be.equal(["10^\u22123", "10^\u22121", "10^0", "10^1", "10^2"])
```

Comment:
Is this format intentional to draw attention to the difference from a an ascii dash? If not, I think just pasting the unicode symbol in the test file would also work.